docs/edit.html

```diff
@@ -28,9 +28,9 @@
             }
 
             if (config.version) {
-                if (config.version === 'master') {
-                    config.cssFile = 'https://leafletjs-cdn.s3.amazonaws.com/content/build/master/leaflet.css',
-                    config.jsFile = 'https://leafletjs-cdn.s3.amazonaws.com/content/build/master/leaflet-src.js'
+                if (config.version === 'main') {
```

Comment:
I am not entirely sure where this version would be set, it seems this state is retrieved from the URL but I am not sure the reference to it has been updated.